Extensions/pokes.css

```diff
@@ -1,73 +1,215 @@
 }
 
-#xkit-loading_pokemon {
+/*
+ * Pokémon list style 
+*/
+.xkit-pokes-lightbox {
+	position: fixed;
+	top: 0px;
+	bottom: 0px;
+	right: 0px;
+	left: 0px; 
+	z-index: 15000;
+	overflow: hidden;
+	background-color: rgba(25,25,25,.92);
+}
+.gender {
+	display: none;
+	float: right;
+	font-family: "segoe ui symbol";
```

Comment:
I never set any specific font on the icons, so the system standard font has been used forever. I didn't think about that when I coded it up.